### ISSUE TITLE
Accessibility - Three Icons Under Header Are Missing ALT Text

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#18]
The three Icons under the header section are missing ALT text. The images names are getting read out not sure if you want to make them background image because the functions of the icons are being defined by the text that follows the images.

**Comment by adriennerlong** [comment]
I've flagged this as a high priority issue. We will ask the developers to address before 9/11.

**Comment by jfc3-dol** [comment]
@adriennerlong,

Thanks, for getting the developers to fix this item. It's important for those using screen readers.

**Comment by jfc3-dol** [comment]
The images are now combined with the text as one item so I'm closing the issue.
